### PR TITLE
Update community.yml

### PR DESCRIPTION
**Pull request by ShorunTech**
permission to delete statistics  block

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: .github/workflows/community.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
           pr-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
           footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord</a> chat and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'
 
-  statistics:
+
     runs-on: ubuntu-latest
```
Comment (eddiejaoude) [inline .github/workflows/community.yml]: Please delete the whole block, `statistics` and all the nest info, up to line 33

Comment (ShorunTech) [reply .github/workflows/community.yml]: Deleted

File: .github/workflows/community.yml [hunk 2]
```diff
@@ -24,10 +24,5 @@ jobs:
...
 
-  statistics:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v1
-      - uses: EddieHubCommunity/gh-action-community/src/statistics@main
-        with:
-          firebase-key: ${{ secrets.FIREBASE_KEY }}
+
+
```
Comment (eddiejaoude) [inline .github/workflows/community.yml]: Looking better 👍 

The file should always end with 1 blank line only that has no characters or spaces on it

**Comment by ShorunTech** [comment]
Thank you Eddie and thank you for sharing this. willing to know more on Open source.

**Comment by eddiejaoude** [comment]
You are welcome! If you delete the extra line we can get this merged

**Review by ShorunTech — COMMENTED** [review]
Extra line deleted

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Comment by ShorunTech** [comment]
Thank you

**Pull request merged**